src/main/content/_assets/js/openliberty.js

```diff
@@ -116,7 +116,15 @@ var openliberty = (function() {
             $("#toc_inner").css("margin-top", "0px");
         }
         $("#toc_indicator").css("margin-top", "0px");
-        $("#code_column").css({"position":"fixed"})
+
+        //handles where the top of the code column should be
+        if (!inSingleColumnView()) {
+            //below the hotspot in single column view
+            $("#code_column").css({"position":"fixed", "top":"0px"})
```

Comment:
Should the comments be switched since the first one is not in single column?

Reply:
Yup 🤦‍♀️ Fixing now.